### DISCUSSION TITLE
StreamedResponse error / headers have already been sent

### DISCUSSION THREAD

**Discussion by thgauthier** [symfony/symfony#49820] [Q&A] [answered]
Hi everyone !
I have followed symfony doc [streaming-a-response](https://symfony.com/doc/current/components/http_foundation.html#streaming-a-response)
Using symfony server (symfony serve)
But first error : EventSource's response has a MIME type ("text/html") that is not "text/event-stream". Aborting the connection.
Solved by adding : `$response->headers->set('Content-Type', 'text/event-stream');`
But now I have the following error :
Failed to start the session because headers have already been sent
![Capture d’écran du 2023-03-26 15-11-45](https://user-images.githubusercontent.com/95478233/227778020-60f3b70e-d494-4323-98c4-bad132f1e371.png)
![Capture d’écran du 2023-03-26 15-12-09](https://user-images.githubusercontent.com/95478233/227778030-3a4ba748-aeff-43f9-b581-acb37e4e2b92.png)
Thanks

**Comment by stefangr** [comment, marked as answer]
The `$response->send()` should not be necessary in this case.

The referenced documentation shows the standalone case (were the framework is not used).

**Reply by thgauthier** [reply]
Thank you very much.
This problem is fixed.
But I still have trouble to console.log the response:
![Capture d’écran du 2023-03-26 22-09-24](https://user-images.githubusercontent.com/95478233/227803087-5eab2231-bc7a-4dca-9366-8205dee294a8.png)
![Capture d’écran du 2023-03-26 22-20-51](https://user-images.githubusercontent.com/95478233/227802993-f21b6432-8b8f-4d08-8cf0-510da4f8a2a1.png)
Any idea ?
